.code-samples.meilisearch.yaml

```diff
@@ -479,3 +491,23 @@ geosearch_guide_sort_usage_2: |-
       "rating:desc",
     });
   client.index("restaurants").search(searchRequest);
+security_guide_search_key_1: |-
+  Client client = new Client(new Config("http://127.0.0.1:7700", "masterKey"));
```

Comment:
https://github.com/meilisearch/documentation/blob/8a965d6ef66d380b0031f4f5d7299e3807f886fb/.code-samples.meilisearch.yaml#L608

Reply:
Oh I totally forgot the Readme

Reply:
But there are many other `masterKey` in the code_samples and in the code we should not take the opportunity to change them all?

Reply:
Should come from another issue open in integration-guide then, it does not concern only this repo, and the "opportunity" during this release does not exist anymore since a lot of repo has already been released 😅

Reply:
But would be a good change!